### PR TITLE
Remove Puck 😭

### PR DESCRIPTION
**Pull request by mendelB**
### What's this PR do?

This pull request removes the Puck library and analytics tracking from Northstar.

### How should this be reviewed?
Should be straightforward! Just make sure I didn't muck anything else up through my tears.

### Any background context you want to provide?
🥃 
We've moved on to tracking with Snowplow and GTM!

### Relevant tickets

References [Pivotal #167633809](https://www.pivotaltracker.com/story/show/167633809).
See https://github.com/DoSomething/phoenix-next/pull/1837

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.
- [ ] If new attributes were added to users, there is a corresponding PR to surface these in Aurora.
- [ ] If new attributes were added to users, then the data team already knows about these changes.

### CODE REVIEW

File: resources/assets/js/utilities/Analytics.js [hunk 1]
```diff
@@ -222,17 +190,16 @@ export function analyzeWithSnowplow(name, category, action, label, data) {
  */
 const sendToServices = (name, category, action, label, data, service) => {
   switch (service) {
-    case 'ga':
+    case 'google':
```
Comment (katiecrane) [inline resources/assets/js/utilities/Analytics.js]: I didn't see any other updates in the code that changed this - are we currently never sending `ga` as the service anywhere?

Comment (mendelB) [reply resources/assets/js/utilities/Analytics.js]: Ah sorry, I didn't provide context for this change -- this is kind of unrelated but brings this method up to par with [Phoenix analytics](https://github.com/DoSomething/phoenix-next/blob/master/resources/assets/helpers/analytics.js#L128)
(And may we merit to see the day when this is consolidated into its own package so we don't need to manually keep these consistent🤞 🙏  )

Comment (katiecrane) [reply resources/assets/js/utilities/Analytics.js]: Perfect, thanks for keeping things consistent 👍 !!

**Comment by mendelB** [comment]
> I feel like I need to apologize for approving this 😭

🙌Class act. 😢

**Pull request merged**